app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -169,18 +176,29 @@ private fun WallpaperThumbnails(
     onSelectWallpaper: (Wallpaper) -> Unit,
 ) {
     Surface(color = FirefoxTheme.colors.layer2) {
-        LazyVerticalGrid(
-            cells = GridCells.Fixed(numColumns),
-            modifier = Modifier.padding(vertical = 30.dp, horizontal = 20.dp)
-        ) {
-            items(wallpapers) { wallpaper ->
-                WallpaperThumbnailItem(
-                    wallpaper = wallpaper,
-                    defaultWallpaper = defaultWallpaper,
-                    loadWallpaperResource = loadWallpaperResource,
-                    isSelected = selectedWallpaper == wallpaper,
-                    onSelect = onSelectWallpaper
-                )
+        Column(modifier = Modifier.padding(vertical = 30.dp, horizontal = 20.dp)) {
```

Comment:
Big sad we have to do a grid manually. I'm assuming wrapping `LazyVerticalGrid` in a `Column` didn't work?

Reply:
It's apparently an [error in Compose](https://stackoverflow.com/questions/66908737/what-is-the-equivalent-of-nestedscrollview-recyclerview-or-nested-recyclerv) to nest `Lazy` layouts within other scrollable layouts. 

Compose 1.1 introduces a [span parameter to items](https://developer.android.com/reference/kotlin/androidx/compose/foundation/lazy/LazyGridScope#item(kotlin.Any,kotlin.Function1,kotlin.Any,kotlin.Function1)) in a `LazyGridScope`. I experimented with that, but it would have also been awkward as the trailing item has a different background color and padding considerations. Plus I didn't want to handle the upgrade just for this

Reply:
I am curious what is the layout we are trying to get here? Got a link to to figma? It's only an error if we're nesting scroll layouts in the same direction.